service/controller/aws/resource_set.go

```diff
@@ -199,8 +199,10 @@ func newResourceSet(config resourceSetConfig) (*controller.ResourceSet, error) {
 			K8sClient:                config.K8sClient.K8sClient(),
 			Logger:                   config.Logger,
 
-			ClusterIPRange: config.ClusterIPRange,
-			Provider:       config.Provider,
+			CalicoAddress:      config.CalicoAddress,
+			CalicoPrefixLength: config.CalicoPrefixLength,
```

Comment:
PR is about coredns, but only I can see these two additional variables, is it okay?

Reply:
coredns needs to know the Calico CIDR. This is the values used now.

```
clusterDNSIP: 172.31.0.10
```

And after the fix.

```
cluster:
  calico:
    CIDR: 172.24.0.0/16
  kubernetes:
    API:
      clusterIPRange: 172.31.0.0/16
    DNS:
      IP: 172.31.0.10
clusterDNSIP: 172.31.0.10
```